tests/utils.py

```diff
@@ -204,26 +204,26 @@ def __init__(self, caplog, index=0, allow_errors=True, ignore=[]):
         self.allow_errors = allow_errors
         self.ignore = ignore
 
-    def _find(self, loggerpart, level, msg, after=0):
+    def _find(self, loggerpart, level, msg, after=0, min_level=1):
```

Comment:
Do you add a docstring about this new parameter? I think we also need some documentation regarding the default value that is used here.